__tests__/Genereal.test.js

```diff
@@ -0,0 +1,7 @@
+describe('Jest Demo Test', () => {
+    test('expect-toBe', () => {
+        expect(1).toBe(1)
+    })
+  })
```

Comment:
Do we need this on the develop branch (since it's just a demo)?  If it's necessary go ahead and push it, if not we can discard it from the commit